[aws] Update inspector data stream for Cloud Detection and Response (CDR) workflow (#14306)
elastic/integrations: packages/aws/_dev/build/docs/inspector.md

# Inspector

> The [Amazon Inspector](https://docs.aws.amazon.com/inspector/) integration collects and parses data from Amazon Inspector [Findings](https://docs.aws.amazon.com/inspector/v2/APIReference/API_ListFindings.html) REST APIs.

Reviewer: @jamiehynds @cpascale43 , we are changing `AWS Inspector` into `Amazon Inspector` as it should be with this change. We are not changing any field names, so the custom fields are still `aws.inspector.*`. 
Let me know if you have any concerns.

For existing users of AWS integration, before upgrading to `4.0.0` please ensure following requirements are met:

1. Users need [Elastic Security solution](https://www.elastic.co/docs/solutions/security) which has requirements documented [here](https://www.elastic.co/docs/solutions/security/get-started/elastic-security-requirements).
2. To use transforms, users must have:
   - at least one [transform node](https://www.elastic.co/docs/deploy-manage/distributed-architecture/clusters-nodes-shards/node-roles#transform-node-role),
   - management features visible in the Kibana space, and
   - security privileges that:
     - grant use of transforms, and
     - grant access to source and destination indices
   For more details on Transform Setup, refer to the link [here](https://www.elastic.co/docs/explore-analyze/transforms/transform-setup)

> 3. Because the latest copy of vulnerabilities is now indexed in two places, i.e., in both source and destination indices, users must anticipate storage requirements accordingly.

Reviewer 2: ```suggestion
3. Because the latest copy of vulnerabilities is now indexed in two places, that is, in both source and destination indices, users must anticipate storage requirements accordingly.
```
The number in the first clause is confusing; is it one copy for each of the vulnerabilities (arguably plural) or is it one copy of the set of vulnerabilities (singular but would be helped by inclusion of "set of")?
